packages/vm/src/modules/file_system_cache.rs

```diff
@@ -46,7 +46,9 @@ use crate::modules::current_wasmer_module_version;
 ///   To work around this, the version was bumped to "v5" here to invalidate these corrupt caches.
 /// - **v6**:<br>
 ///   Version for cosmwasm_vm 1.3+ which adds a sub-folder with the target identier for the modules.
-const MODULE_SERIALIZATION_VERSION: &str = "v6";
+/// - **v7**:<br>
+///   New version because of Wasmer 4 upgrade
```

Comment:
```suggestion
///   New version because of Wasmer 2.3.0 -> 4 upgrade.
///   This internally changes how rkyv is used for module serialization, making compatibility unlikely.
```